File: tests/benchmark/corpus/go/sqli/sqli_concat.go
package main

import (
	"database/sql"
	"net/http"
)

func handler(w http.ResponseWriter, r *http.Request) {
	id := r.URL.Query().Get("id")
	db, _ := sql.Open("sqlite3", "app.db")
	db.Query("SELECT * FROM users WHERE id = " + id)
}

File: tests/benchmark/corpus/java/sqli/SqliConcat.java
import java.sql.*;
import javax.servlet.http.*;

public class SqliConcat extends HttpServlet {
    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Exception {
        String id = req.getParameter("id");
        Connection conn = DriverManager.getConnection("jdbc:sqlite:app.db");
        Statement stmt = conn.createStatement();
        stmt.executeQuery("SELECT * FROM users WHERE id = " + id);
    }
}
